alchemy/logger.py

```diff
@@ -75,7 +76,12 @@ def _run_worker(self):
             except queue.Empty:
                 pass
             if batch:
-                requests.post(self._url, json=batch, headers=headers)
+                while True:
```

Comment:
Hey @balakhonoff,

Thanks for contribution! The point is good, but implementation is not. Look at new release alchemy==20.5 (https://github.com/catalyst-team/alchemy/releases/tag/v20.05) - this issue was fixed here.

Reply:
@dkuryakin I just tried to make it work=) Thanks!

Reply:
@balakhonoff I see! Implementation was not good as final production solution, but was very helpful to show our "point of pain". So thanks again! ;)